doc/guides/getting-started.md

```
@@ -28,12 +28,48 @@ If you code with AI then your AI must read https://github.com/relaystr/ndk/blob/
+Rust toolchain ios:
+
+```bash
+# 64 bit targets (real device & simulator):
+rustup target add aarch64-apple-ios x86_64-apple-ios
+# New simulator target for Xcode 12 and later
+rustup target add aarch64-apple-ios-sim
+# 32 bit targets (you probably don't need these):
+rustup target add armv7-apple-ios i386-apple-ios
+```
+
+## Install
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)